py_zipkin/zipkin.py

```diff
@@ -375,6 +383,50 @@ def update_binary_annotations(self, extra_annotations):
             # the binary annotations for the logging context directly.
             self.logging_context.binary_annotations_dict.update(extra_annotations)
 
+    def add_sa_binary_annotation(
+        self,
+        port=0,
+        service_name='unknown',
+        host='127.0.0.1',
+    ):
+        """Adds a 'sa' binary annotation to the current span.
+
+        'sa' binary annotations are useful for situations where you need to log
+        where a request is going but the destination doesn't support zipkin.
+
+        Note that the span must have 'cs'/'cr' annotations.
+
+        :param port: The port number of the destination
+        :type port: int
+        :param service_name: The name of the destination service
+        :type service_name: str
+        :param host: Host address of the destination
+        :type host: str
+        """
+        if not self.zipkin_attrs or not self.zipkin_attrs.is_sampled:
+            return
+
+        if 'client' not in self.include:
+            # TODO: trying to set a sa binary annotation for a non-client span
+            # should result in a logged error
+            return
+
+        sa_endpoint = create_endpoint(
+            port=port,
+            service_name=service_name,
+            host=host,
+        )
+        sa_binary_annotation = create_binary_annotation(
+            key='sa',
```

Comment:
We don't do this elsewhere in the code (we should), but can you use `zipkin_core.SERVER_ADDR` instead? I think it's a bit clearer